handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -1093,8 +1093,6 @@ public final SSLEngineResult unwrap(
     }
 
     private SSLEngineResult sslReadErrorResult(int err, int bytesConsumed, int bytesProduced) throws SSLException {
-        String errStr = SSL.getErrorString(err);
```

Comment:
This change was just done to eliminate the need to create it when its not needed.